test/extended/util.go

```diff
@@ -0,0 +1,23 @@
+package extended
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/GoogleCloudPlatform/kubernetes/test/e2e"
+)
+
+var testContext = e2e.TestContextType{}
+
+func kubeConfigPath() string {
+	return os.Getenv("KUBECONFIG")
+}
+
+func writeTempJSON(path, content string) error {
```

Comment:
@deads2k there are two use-cases for this... one is storing a runtime.Object to a file and another is storing a string (basically an output from oc...Output()). I added two functions:

writeObjectToFile()
writeToFile()

(the first one does the decoding of runtime.Object and just call the second one at the end.)